src/components/JobDetailCard.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled';
+
+// Temporarily importing JSON here to test code
+import data from '../assets/data.json';
+
+const JobDetailContainer = styled.div`
+  padding: ${props => props.theme.layout.space500};
+  margin: 0 auto;
+  border-radius: 6px;
+  background-color: ${props => props.theme.colors.white};
+  max-width: 730px;
```

Comment:
Let's avoid defining a max-width for this card. Instead, allow it to span the full width of its parent container (i.e., Layout). This approach will make the card more responsive and adaptable to different layouts and content sizes.

Reply:
Good suggestion, adding that change.